mmfunctions

```diff
@@ -1232,20 +1223,19 @@ _prep_volume_adjustment(){
     unset VOLADJ
     if [[ "${VOLADJUST}" = "Y" ]] ; then
         if [[ -f "${INGESTLOG}" ]] ; then
-            VOLADJ=$(_readingestlog "audio_adjustment")
+            measured_I=$(_readingestlog "r128_integrated_loudness")
+            measured_LRA=$(_readingestlog "r128_loudness_range")
+            measured_TP=$(_readingestlog "measured_true_peak")
+            measured_thresh=$(_readingestlog "measured_threshold")
+            target_offset=$(_readingestlog "target_offset")
+            LOUD_ADJ=$(_readingestlog "loudness_adjustment")
         fi
-        if [ "${VOLADJ}" ] ; then
-            _add_audio_filter "volume=${VOLADJ}dB"
+        if [ "${LOUD_ADJ}" ] ; then
+            _add_audio_filter "loudnorm=measured_I="${measured_I}":measured_LRA="${measured_LRA}":measured_TP="${measured_TP}":measured_thresh="${measured_thresh}":offset="${target_offset}":linear=true:print_format=summary"
         else
             _get_volume_adjustment "${SOURCEFILE}"
-            if [ "${VOLADJ}" ] ; then
-                if [ $(echo "${VOLADJ} < 2" |bc) -eq 1 -a $(echo "$VOLADJ > -2" |bc) -eq 1 ] ; then
-                    _report -dt "Integrated loudness for $(basename "${INPUT_MOVIE}") is ${INTEGRATED_LOUDNESS}dB. Reference is ${REFERENCE}dB. No adjustment is needed, skipping."
-                else
-                    _report -dt "Integrated loudness for $(basename "${INPUT_MOVIE}") is ${INTEGRATED_LOUDNESS}dB. Reference is ${REFERENCE}dB. Will adjust by ${VOLADJ}dB."
-                    _add_audio_filter "volume=${VOLADJ}dB"
-                fi
-            fi
+                    _report -dt "Integrated loudness for $(basename "${INPUT_MOVIE}") is ${measured_I} LUFS. Reference is -24 LUFS. Will adjust by ${LOUD_ADJ} LUFS"
+                    _add_audio_filter "loudnorm=measured_I="${measured_I}":measured_LRA="${measured_LRA}":measured_TP="${measured_TP}":measured_thresh="${measured_thresh}":offset="${target_offset}":linear=true:print_format=summary"
```

Comment:
same quoting issue, also i think the logic could be simplified, so that we don't both need 1234 and 1238